docs/user/dashboard/drilldowns.asciidoc

```diff
@@ -40,6 +40,12 @@ For a comparison of the Elastic subscription levels,
 see https://www.elastic.co/subscriptions[the subscription page].
 ==============================================
 
+[NOTE]
```

Comment:
Is this note necessary as the individual pages have a list of supported panels?

Reply:
I removed the note.
I thought it make sense to emphasise that not all panels support drilldown and then that support is different per panel type. But I guess this is indeed redundant.

Reply:
My bigger concern was that there were back-to-back notes, which can be difficult to read.  Another option is to put the text about panel support above the note that starts with "Some action types" .  But don't use Note format for the panel support text.

Reply:
I see. I am good with removing! Thanks for feedback